index.js

```diff
@@ -51,7 +51,7 @@ function paginate(query, options, callback) {
     }
     promises = {
       docs: docsQuery.exec(),
-      count: this.count(query).exec()
+      count: this.estimatedDocumentCount(query).exec()
```

Comment:
Please, change this to `countDocuments(query)` as `estimatedDocumentCount` doesn't accept a query and doesn't guarantee accurate count, this is causing tests to fail and blocking the merge.